test/drenv/test.py

```diff
@@ -101,12 +101,14 @@ def setup():
     Set the environemnt for running the tests.
     """
     _deploy_dr_policy()
+    _deploy_channel()
```

Comment:
This is eliminated in ocm-ramen-samples - the channel will be part of the subscription, so we can remove this code.